src/rpc/client.cpp

```diff
@@ -149,6 +149,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "pruneblockchain", 0, "height" },
     { "keypoolrefill", 0, "newsize" },
     { "getrawmempool", 0, "verbose" },
+    { "getmempoolinfo", 0, "fee_histogram" },
```

Comment:
This shouldn't interrupt `getrawmempool`...

Reply:
Yes, thank you.